src/test/scala/com/cloudera/sparkts/models/ARIMAXSuite.scala

```diff
@@ -54,18 +55,372 @@ class ARIMAXSuite extends FunSuite {
     24,24,25,25,25,25,25,25,25,23,23,23,23,23,23,23,
     51,54,49,46,42,41,45,46,48,41,42,48,43,47,48,46 ))
 
+  def average(v: DenseVector): Double = {
+    var sum = 0.0
+    v.values.map( va => sum += va)
+    sum/tsTrain.values.length
+  }
+  def mse(orginal: BreezeDenseVector[Double], predicted: BreezeDenseVector[Double]): Double = {
+    var sumSquaredError = 0.0
+    for (o <- orginal; p <- predicted){
+      val error = o - p
+      sumSquaredError += Math.pow(error, 2)
+    }
+    val mse = Math.sqrt(sumSquaredError / orginal.length)
+    println(s"MSE =$mse")
```

Comment:
Is there something that we can check with the mse value rather than doing the print?